### PR TITLE
feat: int64/float64 added for AttributeKeyDataType along with validation

### LINKED ISSUE #2540: Add int64/float64 instead of number in AttributeKeyDataType

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes #2540

### CODE REVIEW

File: pkg/query-service/model/v3/v3.go [hunk 1]
```diff
@@ -187,6 +187,15 @@ const (
 	TagTypeResource TagType = "resource"
 )
 
+func (q TagType) Validate() error {
+	switch q {
+	case TagTypeColumn, TagTypeTag, TagTypeResource:
```
Comment (makeavish) [inline pkg/query-service/model/v3/v3.go:192]: @nityanandagohain We don't have TagTypeColumn as it's replaced by isColumn param.

File: pkg/query-service/model/v3/v3.go [hunk 2]
```diff
@@ -267,9 +286,10 @@ func (a AttributeKey) Validate() error {
 }
 
 type FilterAttributeValueResponse struct {
-	StringAttributeValues []string      `json:"stringAttributeValues"`
-	NumberAttributeValues []interface{} `json:"numberAttributeValues"`
-	BoolAttributeValues   []bool        `json:"boolAttributeValues"`
+	StringAttributeValues  []string      `json:"stringAttributeValues"`
+	Int64AttributeValues   []interface{} `json:"int64AttributeValues"`
```
Comment (makeavish) [inline pkg/query-service/model/v3/v3.go:290]: Can't we use numberAttributeValues for both int64 and float64?

Comment (nityanandagohain) [reply pkg/query-service/model/v3/v3.go:290]: If we are building queries using the type then it won't be possible in logs.

File: pkg/query-service/model/v3/v3.go (hunk 1 above)
Comment (nityanandagohain) [reply pkg/query-service/model/v3/v3.go:192]: Sure.

Comment (nityanandagohain) [reply pkg/query-service/model/v3/v3.go]: made the change

File: pkg/query-service/model/v3/v3.go (hunk 2 above)
Comment (nityanandagohain) [reply pkg/query-service/model/v3/v3.go]: made the change

**Review by makeavish — APPROVED** [review]

**Pull request closed without merging**